spec/system/documents/display_and_publish_documents_spec.rb

```diff
@@ -77,6 +75,79 @@
 
         click_button "Save and return"
       end
+
+      it "Assessor is able to add documents to decision notice without publishing" do
+        within(all(".app-task-list__item").first) do
+          click_link "Edit"
+        end
+        fill_in "Document number(s)", with: "new_number_1, new_number_2"
+
+        within(".display") do
+          choose "Yes"
+        end
+
+        within(".publish") do
+          choose "No"
+        end
+
+        click_button "Save and return"
+
+        within(all(".app-task-list__item").first) do
+          click_link "Edit"
+        end
+        # the submitted values are re-presented in the form
+        expect(find_field("numbers")).to have_content "new_number_1, new_number_2"
```

Comment:
Does `expect(page).to have_field("numbers", with: "new_number_1, new_number_2")` work here?

Reply:
Yes! 🚀